### PR TITLE
Add the four simulation types

### PR DESCRIPTION
**Pull request by chadqueen**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+            minimum: 10
+            maximum: 1000
+        scanSpeedValues:
+          description: Array of scan speed values to simulate across, Each value must be between 10 to 10000 mm/s
+          type: array
+          items:
+            type: number
+            minimum: 0.01
+            maximum: 10
+        powderThermalConductivityValues:
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 2]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+          description: Array of Powder Thermal Conductivity Percentage Values to simulate across, Each value must be between 0.1% and 10%
+          type: array
+          items:
+            type: number
+            minimum: 0.001
+            maximum: 0.1
+        powderPackingDensityValues:
+          description: Array of Powder Packing Density Percentage Values to simulate across, Each value must be between 40% and 70%
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 3]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+            minimum: 0
+            maximum: 125000
+        meanFreePathInSolidValues:
+          description: Array of Mean Free Path of Laser in Solid Values to simulate across, Each value must be between 0 and 1.25e5
+          type: array
+          items:
+            type: number
+            minimum: 0
+            maximum: 125000
+        laserAbsorptivityInPowderValues:
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 4]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+          description: Array of Powder Laser Absorptivity Values to simulate across, Each value must be between 50% and 80%
+          type: array
+          items:
+            type: number
+            minimum: 0.5
+            maximum: 0.8
+        laserAbsorptivityInSolidValues:
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 5]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+          description: Array of Solid Laser Absorptivity Values to simulate across, Each value must be between 30% and 60%
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 6]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+            maximum: 125000
+        meanFreePathInSolidValues:
+          description: Array of Mean Free Path of Laser in Solid Values to simulate across, Each value must be between 0 and 1.25e5
+          type: array
+          items:
+            type: number
+            minimum: 0
+            maximum: 125000
+        laserAbsorptivityInPowderValues:
+          description: Array of Powder Laser Absorptivity Values to simulate across, Each value must be between 50% and 80%
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 7]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+            maximum: 10
+        powderThermalConductivityValues:
+          description: Array of Powder Thermal Conductivity Percentage Values to simulate across, Each value must be between 0.1% and 10%
+          type: array
+          items:
+            type: number
+            minimum: 0.001
+            maximum: 0.1
+        powderPackingDensityValues:
+          description: Array of Powder Packing Density Percentage Values to simulate across, Each value must be between 40% and 70%
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

File: swagger.yaml [hunk 8]
```diff
@@ -479,8 +1233,843 @@ definitions:
...
+            maximum: 1000
+        scanSpeedValues:
+          description: Array of scan speed values to simulate across, Each value must be between 10 to 10000 mm/s
+          type: array
+          items:
+            type: number
+            minimum: 0.01
+            maximum: 10
+        powderThermalConductivityValues:
+          description: Array of Powder Thermal Conductivity Percentage Values to simulate across, Each value must be between 0.1% and 10%
```
Comment (pkrull) [inline swagger.yaml]: Change all percent bounds to 0-100%

**Comment by ryanwalls** [comment]
Did you have more to add to this before we think about merging?  Did you validate it at http://editor.swagger.io?

**Pull request merged**